cmake/modules/SwiftHandleGybSources.cmake

```diff
@@ -1,7 +1,32 @@
 include(SwiftAddCustomCommandTarget)
 include(SwiftSetIfArchBitness)
 
-find_package(Python2 COMPONENTS Interpreter REQUIRED)
```

Comment:
This was explicitly `Python2` because this script still requires python2.  Please restore this.

Reply:
That is kind of the point of the whole patch. To use Python2 now or if Python2 does not exist use Python3. I have other patches that make GYB work with systems with only Python3. Once I can prove they work on macOS and Ubuntu 18.04 I was going to submit them here.

I guess it just only makes sense in my head. I will back out 23daa99dff1005b41a657f7cd87b5ce493df1650.